easybuild/easyconfigs/t/tabix/tabix-0.2.6-foss-2019b.eb

```
@@ -0,0 +1,37 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'tabix'
+version = '0.2.6'
+
+homepage = 'http://samtools.sourceforge.net'
+description = """ Generic indexer for TAB-delimited genome position files """
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Tabix can be pushed to a `GCCcore` toolchain as it does not use BLAS/LAPACK or MPI.
```suggestion
toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```

---

@lexming : There is already a GCCcore version, albeit older:
tabix-0.2.6-GCCcore-7.3.0.eb
Do you want me to update that to the latest GCCcore and add this to this PR?

---

You should update this easyconfig to GCCcore version 8.3.0, which is the GCCcore that belongs to foss/2019b. It is not necessary to have any easyconfig of tabix in `foss` as it does not use any of its features.